source/faq.txt

```diff
@@ -232,4 +234,11 @@ Can I customize chunk distributions when syncing into a sharded cluster?
 No, you can't configure ``mongosync`` to customize chunk distributions
 on a destination sharded cluster.  ``mongosync`` samples each collection
 during initialization to determine how to distribute documents
-efficiently across the destination cluster’s shards after migration.
\ No newline at end of file
+efficiently across the destination cluster’s shards after migration.
+
+Can I use ``mongosync`` to set up a Disaster Recovery cluster?
+--------------------------------------------------------------
+
+No, you can't currently set up a Disaster Recovery cluster with ``mongosync``, 
+since ``mongosync`` **must commit** in order to safely accept traffic to the 
```

Comment:
can we link to https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/commit/#std-label-c2c-api-commit for commit